jxl/src/render/simple_pipeline.rs

```diff
@@ -19,6 +19,13 @@ use super::{
     internal::RenderPipelineStageType,
 };
 
+const CACHE_LINE_BYTE_SIZE: usize = 64;
+
+pub fn round_up_size_to_two_cache_lines<T>(size: usize) -> usize {
```

Comment:
this seems a bit out of place here, probably should go to the simd or some util file?

Reply:
you are probably right. done.